devicetypes/smartthings/zigbee-power-meter.src/zigbee-power-meter.groovy

```diff
@@ -58,11 +59,11 @@ def parse(String description) {
                 event.value = event.value/10
                 event.unit = "W"
             } else {
-                event.value = event.value/1000
+                event.value = event.value/divisor
```

Comment:
While you're doing us a favor and getting rid of these magic numbers, could you separate them into something like `powerDivisor` and `energyDivisor`?

Reply:
Not issue Steven, will also replace 10 with `activePowerDivisor` which represents the ActivePower attribute (0x050b) of Electrical measurement cluster (0b05) in the DTH.

Reply:
Not an issue Steven, also replaced 10 with `activePowerDivisor` which represents the ActivePower attribute (0x050b) in the Electrical measurement cluster (0x0b05).